packages/next/shared/lib/router/utils/prepare-destination.ts

```diff
@@ -4,9 +4,14 @@ import type { NextParsedUrlQuery } from '../../../../server/request-meta'
 import type { Params } from '../../../../server/router'
 import type { RouteHas } from '../../../../lib/load-custom-routes'
 import { compile, pathToRegexp } from 'next/dist/compiled/path-to-regexp'
-import escapeStringRegexp from 'next/dist/compiled/escape-string-regexp'
 import { parseUrl } from './parse-url'
 
+// this isn't importing the escape-string-regex module
+// to prevent incompatibility with ie11
+function escapeStringRegexp(str: string) {
```

Comment:
Can we get rid of the package altogether and put this in a separate file?

Reply:
Updated to remove the package